website/docs/sdk-reference/unity.mdx

```diff
@@ -0,0 +1,208 @@
+            switch (level)
+            {
+                case LogLevel.Error:
+                    Debug.LogError(logMessage);
+                    break;
+                case LogLevel.Warning:
+                    Debug.LogWarning(logMessage);
+                    break;
+                case LogLevel.Info:
+                    Debug.Log(logMessage);
+                    break;
+                case LogLevel.Debug:
+                    Debug.Log(logMessage);
+                    break;
+            }
+        }
+    }
+}
+```
+
+:::caution
+In case your application or game targets WebGL, this won't cut it.
+To make the _ConfigCat SDK_ work, you will need additional code, as shown [here](https://github.com/configcat/.net-sdk/blob/master/samples/UnityWebGL/Assets/Scripts/SingletonServices.cs).
+:::
+
+Finally, to activate the `SingletonServices` script, add an empty GameObject to your first scene and assign the script to it.
+
+### 4. Get your setting value
+
+Once the setup above is done, you will be able to access the _ConfigCat_ client in your other scripts and use it to evaluate feature flags and settings:
+
+```cs
+var configCatClient = SingletonServices.Instance.ConfigCatClient;
+
+var isMyAwesomeFeatureEnabled = await configCatClient.GetValueAsync("isMyAwesomeFeatureEnabled", false);
+if (isMyAwesomeFeatureEnabled)
+{
+    doTheNewThing();
+}
+else
+{
+    doTheOldThing();
+}
+```
+
+:::info
+As you can see, feature flag evaluation is asynchronous by default, which in some cases may not play well with Unity's programming model.
+(Might lead to `async void` event handlers or other cumbersome solutions.) If this is the case for you, consider using [non-blocking
+synchronous evaluation with snapshots](./dotnet.mdx#snapshots-and-non-blocking-synchronous-feature-flag-evaluation).
+:::
+
+## Platform compatibility
+
+Proper support for Unity is available since _ConfigCat .NET SDK_ v9.3.0.
+Earlier versions may work with certain build settings (although certainly not with WebGL), but it is recommended to use the latest SDK version.
+
+According to our tests, the _ConfigCat .NET SDK_ works fine with Unity 2021.3 or newer, over both the Mono and IL2CPP [scripting backends](https://docs.unity3d.com/Manual/ScriptingRestrictions.html).
+
+As long as you use the `netstandard2.0` build of the SDK and its dependencies, the "Api Compatibility Level" setting shouldn't matter, both the ".NET Framework" and ".NET Standard 2.1" options should be OK.
+If you experience any issue, try to set the "Api Compatibility Level" setting to ".NET Standard 2.1" on the Edit / Project Settings... / Player tab.
+
+:::info
+We strive to provide an extensive support for the various Unity versions and scripting backends. If you still encounter an issue with the SDK on some platform, please open a [GitHub issue](https://github.com/configcat/.net-sdk/issues/new/choose) or contact support.
```

Comment:
Nekem itt is hiányzik a "contact support" résznél vagy a support email címe, vagy a Contact Us page linkje.

Reply:
Jó észrevétel! 👍 Javítottam.